front-end-master/landing_page/index.html

```diff
@@ -25,8 +25,17 @@
     <link rel="stylesheet" media="screen and (max-device-width: 750px)" href="mobile.css" />
```

Comment:
This works perfectly in the home page but is not implemented in Projects/About Us pages. How can we make this logic modular and reusable?

Reply:
Not my code. Can't say.

Reply:
The user story/work item is to get this working in all header in each page. So we need to get this working every where.

Reply:
the mobile.css here is for the landing page only. I'll just add media queries in the css file of the other pages.